spec/services/ccms/case_add_requestor_spec.rb

```diff
@@ -210,6 +210,11 @@ module CCMS # rubocop:disable Metrics/ModuleLength
                capital_contribution: 0.0
       end
 
+      let(:merits_assessment) do
+        double 'MeritsAssesment',
+               success_prospect: 'Good'
```

Comment:
Why are you using a double, when there is a perfectly good real object to use? Also if this does represent a `MeritsAssessment` there is no "Good" `success_prospect`. I think the value should be "likely". So:
```ruby
let(:merits_assessment) { create :merits_assessment, success_prospect: 'likely' }
```